dev.zed.Zed.metainfo.xml

```diff
@@ -0,0 +1,64 @@
+    </p>
+    <p>
+      Zed leverages every CPU core and your GPU to start instantly, load files in a blink, and respond to your keystrokes on the next display refresh. It supports GitHub Copilot out of the box, and you can use GPT-4 to generate or refactor code with natural language prompts.
+    </p>
+    <p>
+      Zed maintains a full syntax tree for every buffer, enabling precise code highlighting, auto-indent, a searchable outline view, and structural selection. It also supports the Language Server Protocol for autocompletion, code navigation, diagnostics, and refactorings.
+    </p>
+    <p>
+      Zed integrates a full-featured terminal, Vim-style modal editing, and a variety of light and dark themes. Its collaborative features include shared workspaces, real-time navigation, and a virtual office for software teams with channels for discussion, planning, and project tracking.
+    </p>
+    <p>
+      Zed's performance is driven by the GPUI framework, multi-core support with Rust, and extensive fuzz testing for stability. Its CRDT-based buffers provide a foundation for multiple forms of collaboration.
+    </p>
+    <p>
+      NOTE: This wrapper is not yet verified by, affiliated with, or supported by the developers of Zed.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://zed.dev/_next/image?url=%2F_next%2Fstatic%2Fmedia%2Fhero-1.bb06921e.png&amp;w=2048&amp;q=75</image>
+    </screenshot>
+    <screenshot>
+      <image>https://zed.dev/_next/image?url=%2F_next%2Fstatic%2Fmedia%2Flanguage-aware.147d5036.png&amp;w=3840&amp;q=75</image>
+    </screenshot>
+    <screenshot>
+      <image>https://zed.dev/_next/image?url=%2F_next%2Fstatic%2Fmedia%2Fnavigate-together-1.5e8a2161.png&amp;w=3840&amp;q=75</image>
+    </screenshot>
+  </screenshots>
+  <videos>
+    <video>
+      <source>https://customer-snccc0j9v3kfzkif.cloudflarestream.com/b66a1a0a8b375d13c19afe4b37f92562/downloads/default.mp4</source>
+    </video>
+    <video>
+      <source>https://customer-snccc0j9v3kfzkif.cloudflarestream.com/99f0ed130711eb6f47fca1777efa57ef/downloads/default.mp4</source>
+    </video>
+  </videos>
+  <launchable type="desktop-id">dev.zed.Zed.desktop</launchable>
+  <categories>
+    <category>Development</category>
+    <category>Utility</category>
+  </categories>
+  <developer_name>Zed Developers</developer_name>
+  <update_contact>@ZedSupport</update_contact>
+  <url type="homepage">https://www.zed.dev/</url>
+  <url type="bugtracker">https://github.com/zed/zed/issues</url>
+  <url type="help">https://www.zed.dev/support</url>
+  <project_group>
+    <group>Zed</group>
+  </project_group>
+  <releases>
+    <release version="v0.135.2" date="2024-05-15"/>
+  </releases>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <content_rating type="oars-1.1"/>
```

Comment:
Since Zed has audio and chat functionality, this needs to reflect that:

```
<content_attribute id="social-chat">intense</content_attribute>
<content_attribute id="social-audio">intense</content_attribute>
```

Reply:
Added to metainfo. Thanks!